### ISSUE TITLE
Build support for wear OS 

### ISSUE THREAD

**Issue by yourfavDev** [fyne-io/fyne#3754]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same feature, before opening a new one.
- [X] This issue only relates to a single feature. I will open new issues for any other features.

### Is your feature request related to a problem?

I think it would be nice to be able to cross compile my fyne codebase to wear os

### Describe the solution you'd like to see.

A basic implementation at least !

**Comment by andydotxyz** [comment]
Totally would be cool.
Not sure what would be involved in this. Let's discuss possible approaches after we complete the scope for next release...

**Comment by yourfavDev** [comment]
Thank you for creating this milestone, hope we get to use it soon !

**Comment by Bluebugs** [comment]
If you can find documentation on how to target wear OS, that would be very helpful to move this task forward.

**Comment by yourfavDev** [comment]
I could not find anything related to this on the web so that is why i opened this issue, i am afraid the docs does not cover anything like this although it would be useful as not so many apps have a wear os counter part and the demand for such apps is rising

**Comment by yourfavDev** [comment]
Any update?

**Comment by Jacalz** [comment]
No, the issue is still open and it has only been two weeks since you opened it. We will write here in case work is started.

This is scheduled preliminary to the release after the one we currently are working on (see the milestone). There are other issues that need to be prioritised at the moment. It is also worth keeping in mind that this is a complex issue that would be more or less impossible to work on without having hardware to test it on…

**Comment by Bluebugs** [comment]
The lack of documentation or any technical direction to take is also going to impede on the speed of resolving this issue. Figuring how to write open gl application on wear os would really help to move this issue forward for example.
